requirements/base.in

```diff
@@ -1,10 +1,12 @@
 -c constraints.in
-layoutparser[layoutmodels,tesseract]
+layoutparser
```

Comment:
Are we not able to lose `layoutparser` completely?

Reply:
We still have this `TextBox` element we rely on from `layoutparser`, but I wonder if we can just copy that over

https://github.com/Unstructured-IO/unstructured-inference/blob/81549a7e24b66089995bfe7c619f35ee6774bda3/unstructured_inference/inference/layoutelement.py#L7